templates/content_hub_registration_box.mustache

```diff
@@ -26,9 +26,9 @@
                     {{/phone}}
                 </div>
                 <div style="margin-top: 1em">
-                    {{# str}} contenthub:changesettings, mod_hvp, {{ registrationurl }} {{/str}}
+                    {{#changesettings}} contenthub:changesettings, mod_hvp, {{ registrationurl }} {{/changesettings}}
```

Comment:
Same as above here. The {{#str}} name, component {{/str}} is a specific function to get a string at name + component, so its not needed to retain it. Just using {{var}} will put the var in place {{{var}}} with 3 brackets will treat the var as valid HTML, so you can use that to embed divs etc